README.md

```diff
@@ -53,8 +53,16 @@ Set your editor to use spaces, not tabs.
 ### Lowercase all element and attribute names.
 
 
-<a name="for-empty-elements-do-not-use-xhtml-style-closing-syntax--—-just-use-"></a>
-### For "empty" elements, do not use XHTML-style closing syntax `<br />` — just use `<br>`.
+<a name="use-id-attributes-sparingly"></a>
+### Use ID attributes sparingly.
+
+* NEVER use IDs as CSS selectors; it makes style overrides unnecessarily difficult.
+* Don't use IDs for JavaScript hooks: use a "js-" prefixed classname instead.
+* Don't use IDs for automated testing hooks: use a "test-" prefixed classname instead.
```

Comment:
definitely agree, obv, but I sort of thought we had this in another style guide somewhere already?